src/Solver/time_stepping/AbstractTimeCluster.cpp

```diff
@@ -92,8 +93,7 @@ void AbstractTimeCluster::unsafePerformAction(ActorAction action) {
     case ActorAction::Sync:
       assert(state == ActorState::Corrected);
       logDebug(MPI::mpi.rank()) << "synced at " << syncTime
-                                << ", corrTIme =" << ct.correctionTime
-                                << " stepsSinceLastSync " << ct.stepsSinceLastSync
+                                << ", corrTime =" << ct.correctionTime
                                 << " stepsUntilLastSync " << ct.stepsSinceLastSync
```

Comment:
The string says `stepsUntilLastSync`, but the outputted variable says `stepsSinceLastSync`.